File: app/src/main/java/net/bashayer/eticket/qr/models/EventAttendee.java
package net.bashayer.eticket.qr.models;

import java.util.Date;

public class EventAttendee {

    public String eventId;
    public String eventAttendeeName;
    public String eventName;
    public String eventDescription;
    public String city;
    public Date eventDate;

    public EventAttendee(String eventName, String eventAttendeeName) {
        this.eventName = eventName;
        this.eventAttendeeName = eventAttendeeName;
    }

    public EventAttendee(String eventId, String eventAttendeeName, String eventName, String eventDescription, String city, Date eventDate) {
        this.eventId = eventId;
        this.eventAttendeeName = eventAttendeeName;
        this.eventName = eventName;
        this.eventDescription = eventDescription;
        this.city = city;
        this.eventDate = eventDate;
    }
}

File: app/src/main/java/net/bashayer/eticket/qr/models/EventAttendee.kt
package net.bashayer.eticket.qr.models

import android.graphics.Bitmap
import java.util.*

class EventAttendees(val attendees: List<EventAttendee>)
class AttendeeTicket(val QRCodeImage: Bitmap, val eventId: String, val eventAttendeeName: String, val eventName: String, val eventDescription: String, val city: String, val eventDate: Date)